analysis/02_new_model_input/02.0_Building_Damage_README.md

```diff
@@ -0,0 +1,63 @@
+# READ ME for Rasterising Building Damage Data
+
+This section describes the files contained in the building damage input folder.
+`\02_housing_damage\input\`
+
+- `adm3_area.csv`
+  - contains the area of the polygons for the municipalities in the Philippines
+  - exported from QGIS.
+- `Adm3_Perc_inGrid_Weight_Matrix.csv`
+  - shows the percentage of a grid covered by a specific municipality.
+  - This is contained in the Grid Completeness column.
+  - The values for one grid may not always add up to 100%
+    - the grid may be partially overlaying a body of water.
+- `Grid_Matching_Adm3_Max.csv`
+  - shows one version of municipality assignment to
+  - using the municipality that covers the largest area in the grid.
+- `Grid_Perc_inAdm3_Weight_Matrix.csv`
+  - shows the percentage of a municipality that is in a grid.
+  - This is contained in the Municipality Completeness column.
+  - The values for one municipality tend to add up to 100%.
+- `IMpact_data_philipines_SEP_2021.csv`
+  - contains the typhoon impact data by municipality.
+- `phl_adm3_grid_intersections_area.csv`
+  - contains the area of the intersections of the municipalities and the grid
+  - exported from QGIS.
+- `phl_hotosm_bld_centroid_count.csv`
+  - contains the number of building centroids per grid
+  - a file combining output for the whole of the Philippines.
+- `phl_north_hotosm_bld_centroid_count.csv`
+  - contains the number of building centroids per grid
+  - computed from the HOTOSM shapefile for north Philippines.
+- `phl_south_hotosm_bld_centroid_count.csv`
+  - contains the number of building centroids per grid
+  - computed from the HOTOSM shapefile for south Philippines.
+- `typhoons.csv`
+  - contains typhoon names and years
+
+This section describes the files contained in the building damage output folder.
+`\02_housing_damage\output\`
+
+- `percentage_building_damage_bygrid.csv`
+  - contains the output of the rasterisation of the building damage data.
+- `phl_0.1_degree_grid.gpkg`
+  - is the geopackage for the grid overlay as polygons on the Philippines
+  - exported from QGIS.
+- `phl_0.1_degree_grid_centroids.gpkg`
+  - is the geopackage for the centroids of the grid overlay for the Philippines
+  - exported from QGIS.
+- `phl_0.1_degree_grid_centroids_land_overlap.gpkg`
+  - is the geopackage for the centroids of the grid overlay for the Philippines
+  - exported from QGIS.
+  - only covers centroids of grids which overlap with land no matter how small.
+
+The following section contains the processes done in QGIS.
+
+- Based on [this document](https://docs.google.com/document/d/1NprGn6_7gPuf3chmtxig53tGW5vxidM6tDiSOBbIgzM/edit),
```

Comment:
Could you move the documents to [this folder](https://drive.google.com/drive/u/2/folders/1I78yqxY5pPIO-kJik2hRJBdMPbER8x2t) (or a sub-folder)?